nikola/data/samplesite/pages/quickstart.rst

```diff
@@ -363,7 +363,7 @@ Images
 
 __ ../quickref/#directives
 
-To include an image in your document, you use the the ``image`` directive__.
+To include an image in your document, you use the ``image`` directive__.
```

Comment:
This is also a third-party document, this typo fix might be reported upstream if you wish.

Reply:
ok will do